charts/gha-runner-scale-set/values.yaml

```diff
@@ -86,11 +86,11 @@ template:
   ## template:
   ##   spec:
   ##     initContainers:
-  ##     - name: initExternalsInternalVolume
```

Comment:
update the comments to match what gets generated by the helm chart.

Reply:
We should avoid examples with camel case in the future because it is not acceptable by k8s anyway